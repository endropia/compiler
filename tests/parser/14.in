procedure (var b: Integer; const c: Real);
begin
    b := c;
end;