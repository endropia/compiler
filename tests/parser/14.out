(1, 11) Identifier expected